RecoEcal/EgammaClusterProducers/python/multi5x5SuperClusters_cfi.py

```diff
@@ -39,10 +39,9 @@
 )
 
 
-multi5x5SuperClustersUncleaned = multi5x5SuperClustersCleaned.clone()
-multi5x5SuperClustersUncleaned.endcapClusterProducer = cms.string('multi5x5BasicClustersUncleaned')
-multi5x5SuperClustersUncleaned.endcapClusterProducer = cms.string('multi5x5BasicClustersUncleaned')
-
+multi5x5SuperClustersUncleaned = multi5x5SuperClustersCleaned.clone(
+    endcapClusterProducer = cms.string('multi5x5BasicClustersUncleaned')
```

Comment:
"string" can be dropped

Reply:
No, forget it @jeongeun : the corresponding parameter is not listed in the cloned config

Reply:
Triggered by this finding, I'm realizing that a `Multi5x5SuperClusterProducer` does not require a parameter called "endcapClusterProducer" (which would be needed by a `HiSuperClusterProducer`, instead).

If I'm not wrong, one should have replaced here instead

```PYTHON
    endcapClusterTag= cms.InputTag('multi5x5BasicClustersUncleaned',
				   'multi5x5EndcapBasicClusters'),

```

@afiqaize @SohamBhattacharya could you please have a look and tell us if there is a bug here?